src/Congo.WebApi/Controllers/ProductsController.cs

```diff
@@ -28,6 +28,17 @@ public async Task<IEnumerable<Product>> Get()
             return await _mediator.Send(new GetProductListQuery());
         }
 
+
+        [HttpPost]
+        [Route("~/api/{seller}/products")]
```

Comment:
While this works, this will get messy down the line when we start adding more and more "seller" endpoints. I suggest moving the seller product stuff to its own controller, instead of reusing the customer one.